plugins/modules/user_role_workflow_manager.py

```diff
@@ -1056,28 +1056,27 @@ def valid_user_config_parameters(self, user_config):
         """
         self.log("Validating user configuration parameters...", "INFO")
         error_messages = []
+        regex_name_validation = re.compile(r"^[A-Za-z0-9_-]+$")
+        regex_name_validation_msg = "must only contain letters, numbers, underscores and hyphens and should not contain spaces or other special characters."
 
-        first_name_regex = re.compile(r"^[A-Za-z0-9_-]+$")
-        self.validate_string_field(user_config.get("first_name"), first_name_regex, "first_name: 'first_name' must only contain letters, numbers, \
-                                   underscores, and hyphens and should not contain spaces or other special characters.", error_messages)
+        self.validate_string_field(user_config.get("first_name"), regex_name_validation,
```

Comment:
```
        self.validate_string_field(user_config.get("first_name"), regex_name_validation,
                                   "first_name: '{0}' {1}".format(user_config.get("first_name"), regex_name_validation_msg), error_messages)
```

Instead of getting first_name using 'user_config.get("first_name")' two times, we can store and use it.

```
        first_name = user_config.get("first_name")
        self.validate_string_field(first_name, regex_name_validation,
                                   "first_name: '{0}' {1}".format(first_name), regex_name_validation_msg), error_messages)
```

Reply:
Updated the code.